packages/reports/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@opensrp/reports",
+  "version": "0.0.0",
+  "description": "Download OpenSRP reports",
+  "main": "dist/index.js",
+  "types": "dist/types",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/opensrp/web",
+  "config": {
+    "folderName": "reports"
+  },
+  "scripts": {
+    "test": " cd ../../ && yarn test packages/$npm_package_config_folderName --verbose --collectCoverage=true && cd packages/$npm_package_config_folderName",
+    "tsc": "tsc",
+    "lint": "eslint ./**/*.{js,jsx,ts,tsx}",
+    "copy": "copyfiles -u 1 \"./src/**/*.{css,html}\" \"./dist/\"",
+    "build": "yarn run tsc && yarn run transpile && yarn run copy",
+    "transpile": "babel src -d dist --root-mode upward --extensions .ts,.tsx --ignore '**/*.test.ts,**/*.test.tsx,**/tests,**/__tests__'",
+    "extractTranslations": "yarn i18next 'src/**/*.{ts,tsx}' -c ../../i18next-parser.config.js"
+  },
+  "bugs": {
+    "url": "https://github.com/opensrp/web/issues"
+  },
+  "peerDependencies": {
+    "i18next": "^21.6.14",
+    "react": "^16.18.1"
+  },
+  "author": "OpenSRP Engineering",
+  "license": "Apache-2.0",
+  "private": false,
+  "dependencies": {
+    "@ant-design/icons": "^4.7.0",
+    "@onaio/utils": "^0.0.1",
+    "@opensrp/pkg-config": "^0.0.9",
+    "@opensrp/react-utils": "^0.0.12",
+    "@opensrp/server-service": "^0.0.17",
+    "antd": "^4.19.2",
+    "moment": "^2.29.1",
+    "react-query": "^3.34.16",
```

Comment:
different react-queryies might require different Providers, I would vote we move it to peer dependency. antd as well.

Reply:
22bf6a57da5a352b9d78cc49f8c0a4bcbdcabbf6